sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/package-info.java

```diff
@@ -2,7 +2,179 @@
 // Licensed under the MIT License.
 
 /**
- * Package containing classes for creating a {@link com.azure.ai.formrecognizer.FormRecognizerClient} to
- * perform operations on Azure Form Recognizer
+ * <p>The Azure Form Recognizer library provides
```

Comment:
The review comments for file `src/main/java/com/azure/ai/formrecognizer/documentanalysis/package-info.java`
can be mapped idiomatically here as well.